2-structured-data/test/_api-tests.js

```diff
@@ -13,57 +13,52 @@
 
 'use strict';
 
-const getRequest = require(`@google-cloud/nodejs-repo-tools`).getRequest;
-const test = require(`ava`);
+const getRequest = require('@google-cloud/nodejs-repo-tools').getRequest;
+const assert = require('assert');
 
 module.exports = DATA_BACKEND => {
   let originalDataBackend, id, testConfig, appConfig;
 
-  test.before(() => {
-    testConfig = require(`./_test-config`);
-    appConfig = require(`../config`);
-    originalDataBackend = appConfig.get(`DATA_BACKEND`);
-    appConfig.set(`DATA_BACKEND`, DATA_BACKEND);
+  before(() => {
+    testConfig = require('./_test-config');
+    appConfig = require('../config');
+    originalDataBackend = appConfig.get('DATA_BACKEND');
+    appConfig.set('DATA_BACKEND', DATA_BACKEND);
   });
 
-  test.serial.cb(`should create a book`, t => {
-    getRequest(testConfig)
-      .post(`/api/books`)
-      .send({title: `beep`})
+  after(() => {
+    appConfig.set('DATA_BACKEND', originalDataBackend);
```

Comment:
Wondering what this is for

Reply:
DataBackend can be 'datastore' or 'cloudsql'. Be sure to configure the appropriate settings for each storage engine. If you are unsure, use datastore as it requires no additional configuration.
So default is 'datastore' and we set 'cloudsql' and do our work and at the end we set back to default 'datastore'.